wallet/src/de/schildbach/wallet/ui/AbstractBindServiceActivity.java

```diff
@@ -28,15 +28,23 @@
 import android.content.ServiceConnection;
 import android.os.IBinder;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 /**
  * @author Andreas Schildbach
  */
 public abstract class AbstractBindServiceActivity extends AbstractWalletActivity {
 
+    private static final Logger log = LoggerFactory.getLogger(BlockchainServiceImpl.class);
```

Comment:
`AbstractBindServiceActivity` instead of `BlockchainServiceImpl`

Reply:
Thanks Eric!
Fixed.